### ISSUE TITLE
'rpm -Uvh' should not be used for upgrading the builds. 

### ISSUE THREAD

**Issue by ashahi1** [vmware/docker-volume-vsphere#984]
Steps:
1. Installed 0.11.1 build on docker vm and esx.
2. Downloaded and upgraded the builds to 0.12.
```
root@photon-xVmYMbyTn [ /tmp ]# ls
docker-volume-vsphere-0.12.3f64805-1.x86_64.rpm  systemd-private-7e70dce8a837484da28dc3ef7e39a2d9-systemd-timesyncd.service-8AdaoI  vgauthsvclog.txt.0  vmware-root
root@photon-xVmYMbyTn [ /tmp ]# rpm -Uvh docker-volume-vsphere-0.12.3f64805-1.x86_64.rpm
Preparing...                          ################################# [100%]
Updating / installing...
   1:docker-volume-vsphere-0:0.12.3f64################################# [ 50%]
  File: '/proc/1/exe' -> '/usr/lib/systemd/systemd'
  File: '/proc/1/exe' -> '/usr/lib/systemd/systemd'
Removed symlink /etc/systemd/system/multi-user.target.wants/docker-volume-vsphere.service.
Cleaning up / removing...
   2:docker-volume-vsphere-0:0.11.1.f1################################# [100%]
  File: '/proc/1/exe' -> '/usr/lib/systemd/systemd'
```
3. After upgrading, docker-volume-vsphere service is not running:
```
root@photon-xVmYMbyTn [ ~ ]# systemctl status docker-volume-vsphere
● docker-volume-vsphere.service - "Docker Volume Driver for vSphere"
   Loaded: loaded (/usr/lib/systemd/system/docker-volume-vsphere.service; disabled; vendor preset: enabled)
   Active: inactive (dead)

Mar 01 20:28:41 photon-xVmYMbyTn docker-volume-vsphere[716]: rflags 0x206
Mar 01 20:28:41 photon-xVmYMbyTn docker-volume-vsphere[716]: cs     0x33
Mar 01 20:28:41 photon-xVmYMbyTn docker-volume-vsphere[716]: fs     0x0
Mar 01 20:28:41 photon-xVmYMbyTn docker-volume-vsphere[716]: gs     0x0
Mar 01 20:28:41 photon-xVmYMbyTn systemd[1]: docker-volume-vsphere.service: Service hold-off time over, scheduling restart.
Mar 01 20:28:41 photon-xVmYMbyTn systemd[1]: Stopped "Docker Volume Driver for vSphere".
Mar 01 20:28:41 photon-xVmYMbyTn systemd[1]: Started "Docker Volume Driver for vSphere".
Mar 01 20:48:07 photon-xVmYMbyTn systemd[1]: Started "Docker Volume Driver for vSphere".
Mar 01 20:48:07 photon-xVmYMbyTn systemd[1]: Stopping "Docker Volume Driver for vSphere"...
Mar 01 20:48:07 photon-xVmYMbyTn systemd[1]: Stopped "Docker Volume Driver for vSphere".
```

//CC @pdhamdhere

**Comment by pdhamdhere** [comment]
@ashahi1 Can you please confirm if Debian (dpkg) has same behavior with upgrade?

CC/ @kerneltime

**Comment by ashahi1** [comment]
@pdhamdhere Yes, I see the same behavior Ubuntu 14.04.4 LTS.
```
root@sc-rdops-vm02-dhcp-52-237:~# docker volume ls
list vmdk: Post http://%2Frun%2Fdocker%2Fplugins%2Fvmdk.sock/VolumeDriver.List: dial unix /run/docker/plugins/vmdk.sock: connect: no such file or directory
DRIVER              VOLUME NAME
vmdk                vol34
vsphere             vol34@sharedVmfs-0
```

**Comment by pdhamdhere** [comment]
Docker looking for VMDK socket is expected. On Photon upgrade didn't start service. Did it start on Ubuntu? After restarting docker, does 'Docker volume ls' list volumes?

**Comment by pdhamdhere** [comment]
We may not need to fix this after #886

**Comment by pdhamdhere** [comment]
Starting 0.13 we recommend managed docker volume plugin.
